engine/src/flutter/shell/platform/common/accessibility_bridge_unittests.cc

```diff
@@ -241,9 +243,11 @@ TEST(AccessibilityBridgeTest, DoesNotAssignEditableRootToSelectableText) {
   std::shared_ptr<TestAccessibilityBridge> bridge =
       std::make_shared<TestAccessibilityBridge>();
   FlutterSemanticsNode2 root = CreateSemanticsNode(0, "root");
-  root.flags = static_cast<FlutterSemanticsFlag>(
-      FlutterSemanticsFlag::kFlutterSemanticsFlagIsTextField |
-      FlutterSemanticsFlag::kFlutterSemanticsFlagIsReadOnly);
+  FlutterSemanticsFlags flags = FlutterSemanticsFlags{0};
```

Comment:
do we need to call the constructor? I think just
```
FlutterSemanticsFlags flags;
```
will be fine?

Reply:
also why 0?

Reply:
I'm calling the constructor here to make sure the flags are all initialized to false  by default.

Reply:
A little more idiomatic:
```cc
FlutterSemanticsFlags flags = FlutterSemanticsFlags{};
```

And yes; as @hannah-hyj says, this is important since locals are uninitialised in C++, so without this, it'll be populated with whatever memory happens to be on the stack at the time.